### PR TITLE
(COURSES) : Added courses links for the Marathi language

### PR DESCRIPTION
**Pull request by Atharva1723**
## What does this PR do?
Created a new markdown file for the new language Marathi.Added links to programming free courses
## For resources
Youtube playlists
### Why is this valuable (or not)?
Yes, this is valuable as the free courses for Marathi languages were not added before.
### How do we know it's really free?
You can test those links 
### For book lists, is it a book? For course lists, is it a course? etc.
The markdown file I created is for courses
## Checklist:
- [ ✓ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ✓ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ✓ ] Include author(s) and platform where appropriate.
- [ ✓ ] Put lists in alphabetical order, correct spacing.
- [ ✓ ] Add needed indications (PDF, access notes, under construction).
- [ ✓ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Atharva1723** [comment]
@eshellman I have added one more Indian language Marathi. Please review this pr and let me know if there are changes required!

File: courses/free-courses-mr.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+### Databases
+
+* [Database Management System](https://youtube.com/playlist?list=PLNUHhIfQzCNcVcVbMDI8jmjxbZ9u3QSUY) - Dnyaneshwar Cholke
+* [SQL Tutorial For Beginners in Marathi | SQL Structured Query Language Full Course in Marathi Basics To Advance](https://youtube.com/playlist?list=PLFwH5aoadVcnimSkNWYKUjsvOTXvVnfVw) - Code Marathi
+
+
+### HTML and CSS  
+
+* [CSS Tutorial In Marathi](https://youtube.com/playlist?list=PLWkJQ8CSXYd4wee103RY961OdWXwnHsBo) - SA Infolines
+* [HTML | HTML For Beginners | Krushna Pise | Marathi Programmer](https://youtube.com/playlist?list=PLWSZr_wlNWaw8_iFhKvrPKp1Uh2S1dXHk) - MaRaTHi ProGrammer
```
Comment (eshellman) [inline courses/free-courses-mr.md:64]: escape the `|`

**Review by eshellman — CHANGES_REQUESTED** [review]
nice job, just one nit

**Comment by Atharva1723** [comment]
@eshellman I have made the necessary changes

File: courses/free-courses-mr.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+
+### Android
+
+* [अँड्रॉइड ऍप डेव्हलपमेंट मराठी \| Android App Development in Marathi](https://youtube.com/playlist?list=PLy1NOWTMDFjDLCgmxRNLDJtlutJOYl2d7) - उत्तर
+
+
+### Angular
+
+* [Angular Tutorials in Marathi](https://youtube.com/playlist?list=PLMs908ICeVMEBw4CevNdPvvIJDaQV1ISN) - Developers Marathi
```
Comment (eshellman) [inline courses/free-courses-mr.md:24]: sorry, one more nit. could you switch the order of the Angular course so it's alphabetical? thanks

**Pull request closed without merging**